### ISSUE TITLE
hidden files in /etc/netclient

### ISSUE THREAD

**Issue by mattkasun** [gravitl/netmaker#31]
It is very unusual to have hidden files in /etc
Suggest .netconfig and .nettoken change to netconfig and nettoken

```
kasun@aws~/sandbox/netmaker/netclient> sudo find /etc/ -name "\.*" -print
/etc/cron.daily/.placeholder
/etc/cron.hourly/.placeholder
/etc/.pwd.lock
/etc/cron.monthly/.placeholder
/etc/netclient/.netconfig
/etc/netclient/.nettoken
/etc/cron.d/.placeholder
/etc/cron.weekly/.placeholder
/etc/skel/.bash_logout
/etc/skel/.bashrc
/etc/skel/.profile
```
It is also unusual to have an executable (netclient) in /etc in addition to the one in /usr/local/bin

**Comment by afeiszli** [comment]
I agree on netconfig, but nettoken is a JWT used for authorization. Might having it be a hidden file be slightly better?

**Comment by afeiszli** [comment]
fixed with PR #84
